sktime/forecasting/conditional_invertible_neural_network.py

```diff
@@ -208,7 +208,19 @@ def _fit(self, y, fh, X=None):
         Returns
         -------
         self : reference to self
+        Common Errors:
+        --------------
+        1. If `window_size` > len(y_train), an error will be raised.
+        2. If curve fitting fails, ensure your data and window_size are suitable.
+
         """
+        # Ensure that window_size is smaller than the training data
+        forecaster = CINNForecaster(
+            window_size=20
+        )  # Example: window_size < len(y_train)
+
+        forecaster.fit(y)
```

Comment:
Where is the use of this fitted forecaster instance? I couldn't find a reference to this below.